### PR TITLE
React router portfolio branch

### PR DESCRIPTION
**Pull request by oscar-leung**
Source - https://reactrouter.com/docs/en/v6/getting-started/tutorial

Use browser bar 
localhost:3000/
localhost:3000/portfolio
localhost:3000/resume
localhost:3000/blog
localhost:3000/contact-me

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,46 +4,48 @@ import { Button, Link } from "@mui/material";
 import NavBar from "./components/NavBar";
 import HomePage from "./pages/HomePage";
 import {
+  BrowserRouter,
   BrowserRouter as Router,
```
Comment (josekim) [inline src/App.js:8]: duplicate

File: src/App.js [hunk 2]
```diff
@@ -4,46 +4,48 @@ import { Button, Link } from "@mui/material";
...
-        >
-          Learn React
-        </a>
-        <Button variant="contained"> Hello world!</Button>
-        <NavBar></NavBar>
-        {/* <HomePage></HomePage> */}
-        {/* <BlogPage /> */}
+    <ThemeProvider theme={theme}>
+      <BrowserRouter>
+        <div className="App">
```
Comment (josekim) [inline src/App.js:22]: make this a component

File: src/components/NavBar.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ export default function NavBar() {
     setAnchorEl(null);
   };
   return (
-    <div>
+    <React.Fragment>
```
Comment (josekim) [inline src/components/NavBar.js:17]: you can use <>  and </>

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -1,16 +1,7 @@
...
-const Root = styled("div")(({ theme }) => ({
-  width: "100%",
-  ...theme.typography.body2,
-  "& > :not(style) + :not(style)": {
-    marginTop: theme.spacing(2),
-  },
-}));
 
 export default function HomePage() {
   const content = <div>{`This is the home page`}</div>;
```
Comment (josekim) [inline src/pages/HomePage.js:4]: you probably don't need a div inside a h1

File: src/pages/ResumePage.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+export default function ResumePage() {
+  const content = <div>{`This is the resume page`}</div>;
```
Comment (josekim) [inline src/pages/ResumePage.js:4]: dont' need a div inside a h1

File: src/App.js (hunk 2 above)
Comment (oscar-leung) [reply src/App.js:22]: What do you mean by making this a component?

**Pull request merged**